dev/Common/PropertyHelper.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation and Contributors. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+namespace winrt
+{
+    template<typename TStored> struct basic_property
```

Comment:
Can you use [wil::single_threaded_property](https://github.com/microsoft/wil/blob/6f60a1b76fb812c6af5db1bc7abdec0001edd43f/include/wil/cppwinrt_authoring.h#L48) instead of defining a new simple property wrapper?

Reply:
Sure, on the docs, it mentions "For settable properties, use single_threaded_rw_property<T> instead."
I understand that using this is preferable to creating a new property wrapper, does it offer any additional benefits as well?

Reply:
Updated!